lightning/common/security.go

```diff
@@ -141,3 +148,15 @@ func (tc *TLS) WrapListener(l net.Listener) net.Listener {
 func (tc *TLS) GetJSON(path string, v interface{}) error {
 	return GetJSON(tc.client, tc.url+path, v)
 }
+
+func (tc *TLS) ToPDSecurityOption() pd.SecurityOption {
+	return pd.SecurityOption{
+		CAPath:   tc.caPath,
+		CertPath: tc.certPath,
+		KeyPath:  tc.keyPath,
+	}
+}
+
+func (tc *TLS) TransToTlsConfig() (*tls.Config, error) {
```

Comment:
both us and BR should probably change to use https://github.com/pingcap/tidb-tools/blob/master/pkg/utils/security.go, so we can get rid of this function.